tests/issues/issue-2008.php

```diff
@@ -0,0 +1,176 @@
+            'Version' => '44.0',
+            'Platform' => 'macOS',
+            'Platform_Version' => '10.13',
+            'Platform_Description' => 'macOS',
+            'Platform_Bits' => '32',
+            'Platform_Maker' => 'Apple Inc',
+            'Alpha' => false,
+            'Beta' => false,
+            'Frames' => true,
+            'IFrames' => true,
+            'Tables' => true,
+            'Cookies' => true,
+            'JavaScript' => true,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'isSyndicationReader' => false,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '3',
+            'Device_Name' => 'Macintosh',
+            'Device_Maker' => 'Apple Inc',
+            'Device_Type' => 'Desktop',
+            'Device_Pointing_Method' => 'mouse',
+            'Device_Code_Name' => 'Macintosh',
+            'Device_Brand_Name' => 'Apple',
+            'RenderingEngine_Name' => 'Blink',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'Google Inc',
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true,
+    ],
+    'issue-2008-B' => [
+        'ua' => 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.89 Safari/537.36',
+        'properties' => [
+            'Comment' => 'Chrome 44.0',
+            'Browser' => 'Chrome',
+            'Browser_Type' => 'Browser',
+            'Browser_Bits' => '32',
+            'Browser_Maker' => 'Google Inc',
+            'Browser_Modus' => 'unknown',
+            'Version' => '44.0',
+            'Platform' => 'macOS',
+            'Platform_Version' => '10.14',
+            'Platform_Description' => 'macOS',
+            'Platform_Bits' => '32',
+            'Platform_Maker' => 'Apple Inc',
+            'Alpha' => false,
+            'Beta' => false,
+            'Frames' => true,
+            'IFrames' => true,
+            'Tables' => true,
+            'Cookies' => true,
+            'JavaScript' => true,
+            'VBScript' => false,
+            'JavaApplets' => false,
+            'isSyndicationReader' => false,
+            'isFake' => false,
+            'isAnonymized' => false,
+            'isModified' => false,
+            'CssVersion' => '3',
+            'Device_Name' => 'Macintosh',
+            'Device_Maker' => 'Apple Inc',
+            'Device_Type' => 'Desktop',
+            'Device_Pointing_Method' => 'mouse',
+            'Device_Code_Name' => 'Macintosh',
+            'Device_Brand_Name' => 'Apple',
+            'RenderingEngine_Name' => 'Blink',
+            'RenderingEngine_Version' => 'unknown',
+            'RenderingEngine_Maker' => 'Google Inc',
+        ],
+        'lite' => false,
+        'standard' => true,
+        'full' => true,
+    ],
+    'issue-2008-C' => [
```

Comment:
You can remove this test though, we don't typically add tests for the Generic patterns unless it's because we want to make sure invalid versions/platforms aren't added in the future.